kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -86,17 +86,22 @@ if [ "$cloud_mode" = true ]; then
 
     docker_image="us-docker.pkg.dev/linera-io-dev/linera-docker-repo/linera-test-local:latest"
     opt_list+=" --cloud"
-    
+
     docker pull $docker_image || exit 1
 else
     docker_image="linera-test:latest"
     if [ "$do_build" = true ]; then
-        if [ "$copy" = true ]; then
-            docker build -f ../../docker/Dockerfile.copy ../../ -t $docker_image || exit 1
-        elif [ "$(uname -m)" = "x86_64" ]; then
-            docker build -f ../../docker/Dockerfile.local ../../ -t $docker_image || exit 1
+        if [[ "${copy-}" ]]; then
+            docker build \
+                -f ../../docker/Dockerfile.copy \
+                --build-arg environment=k8s-local \
+                ../../
         else
-            docker build -f ../../docker/Dockerfile.local-aarch64 ../../ -t $docker_image || exit 1
+            docker build \
+                -f ../../docker/Dockerfile \
+                ../../ \
+                --build-arg environment=k8s-local \
+                --build-arg target="$(uname -m)-unknown-linux-gnu"
```

Comment:
You're probably gonna get to this (as it's still a draft), but CI is failing because you need to keep the `-t` parameter here specifying the docker image, I believe

Reply:
Oh yeah I didn't mean to drop it :sweat_smile: Just a typo on my part.